raster/r.colors.out/local_proto.h

```diff
@@ -0,0 +1,10 @@
+#include <grass/gis.h>
+#include <grass/parson.h>
+
+enum OutputFormat { PLAIN, JSON };
+
+void write_json_rule(DCELL *val, DCELL *min, DCELL *max, int r, int g, int b,
+                     JSON_Array *root_array, int perc);
+
+void Rast_json_print_colors(struct Colors *colors, DCELL min, DCELL max,
+                            FILE *fp, int perc);
```

Comment:
This is library function naming convention, so it may be confusing here. I don't think we are ready to put this to the library, so let's change it to print_json_colors for this PR.